Gems/Atom/RHI/Vulkan/Code/Source/RHI/ImageView.cpp

```diff
@@ -118,16 +118,26 @@ namespace AZ
                                     viewDescriptor.m_aspectFlags != RHI::ImageAspectFlags::Depth &&
                                     viewDescriptor.m_aspectFlags != RHI::ImageAspectFlags::Stencil;
 
-            if (!viewDescriptor.m_isArray && !viewDescriptor.m_isCubemap && !isDSRendertarget)
+            if (!viewDescriptor.m_isArray && !isDSRendertarget)
             {
-                if (RHI::CheckBitsAll(image.GetDescriptor().m_bindFlags, RHI::ImageBindFlags::ShaderRead))
+                if (!viewDescriptor.m_isCubemap)
```

Comment:
It reads better if you flip the if and else part. 
if (viewDescriptor.m_isCubemap)
...
else
...

Reply:
I was trying to keep it similar to the original check.